src/coreComponents/python/modules/hdf5_wrapper_package/hdf5_wrapper/wrapper.py

```diff
@@ -44,127 +43,127 @@ def __getitem__(self, k):
-        tmp = decode(tmp)
+            # Decode any string types
+            if tmp.dtype.kind in ["S", "U", "O"]:
+                tmp = decode(tmp)
 
-      # Convert any 0-length arrays to native types
-      if not tmp.shape:
-        tmp = tmp[()]
+            # Convert any 0-length arrays to native types
+            if not tmp.shape:
+                tmp = tmp[()]
 
-      return tmp
-    else:
-      return tmp
+            return tmp
+        else:
+            return tmp
 
-  def __setitem__(self, k, value):
-    """
+    def __setitem__(self, k, value):
+        """
     @brief write an object to the database if write-mode is enabled
     @param k the name of the object
     @param value the object to be written
     """
 
-    if (self.mode in ['w', 'a']):
-      if isinstance(value, dict):
-        # Recursively add groups and their children
-        if (k not in self.target):
-          self.target.create_group(k)
-        new_group = self[k]
-        for x in value:
-          new_group[x] = value[x]
-      else:
-        # Delete the old copy if necessary
-        if (k in self.target):
-          del(self.target[k])
-
-        # Add everything else as an ndarray
-        tmp = np.array(value)
-        if (tmp.dtype.kind in ['S', 'U', 'O']):
-          tmp = encode(tmp)
-        self.target[k] = tmp
-    else:
-      raise ValueError('Cannot write to an hdf5 opened in read-only mode!  This can be changed by overriding the default mode argument for the wrapper.')
-
-  def link(self, k, target):
-    """
+        if self.mode in ["w", "a"]:
+            if isinstance(value, dict):
+                # Recursively add groups and their children
+                if k not in self.target:
+                    self.target.create_group(k)
+                new_group = self[k]
+                for x in value:
+                    new_group[x] = value[x]
+            else:
+                # Delete the old copy if necessary
+                if k in self.target:
+                    del self.target[k]
+
+                # Add everything else as an ndarray
+                tmp = np.array(value)
+                if tmp.dtype.kind in ["S", "U", "O"]:
+                    tmp = encode(tmp)
+                self.target[k] = tmp
+        else:
+            raise ValueError(
+                "Cannot write to an hdf5 opened in read-only mode!  This can be changed by overriding the default mode argument for the wrapper."
+            )
+
+    def link(self, k, target):
+        """
     @brief link an external hdf5 file to this location in the database
     @param k the name of the new link in the database
     @param target the path to the external database
     """
-    self.target[k] = h5py.ExternalLink(target, '/')
+        self.target[k] = h5py.ExternalLink(target, "/")
 
-  def keys(self):
-    """
+    def keys(self):
+        """
     @brief get a list of groups and arrays located at the current level
     @return a list of strings 
     """
```

Comment:
Black doesn't check the indentation of multi-line docstrings after the first line, as that could unexpectedly modify its contents